apps/dashboard/src/app/(dashboard)/dashboard/connect/ecosystem/types.ts

```diff
@@ -0,0 +1,22 @@
+export type Ecosystem = {
+  name: string;
+  imageUrl?: string;
+  id: string;
+  slug: string;
+  permission: "PARTNER_WHITELIST" | "ANYONE";
```

Comment:
language, probably needs a backend change?

Reply:
Yes and I'm not sure how set in stone it is yet (I know we do have customers live with ecosystem wallets right now)

Reply:
lets _try_ to get it changed if we can, better now than after this is live